### PR TITLE
Create temp_guide_to_setup_a_node_validator.md

### PR DESCRIPTION
**Pull request by educlerici-zondax**
Create temp_guide_to_setup_a_node_validator.md

### CODE REVIEW

File: temp_guide_to_setup_a_node_validator.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Setting Up a Node/Validator for Paseo Testnet
+
+Welcome to the Paseo testnet! This guide will walk you through the process of setting up a node and becoming a validator on the Paseo testnet network using the Polkadot binary, which is also used for Polkadot and Kusama validators.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A server with a recommended minimum of 8GB RAM, a quad-core CPU, and at least 200GB of free disk space.
+- The latest version of the Polkadot binary installed on your server. You can download it from [the official Polkadot repository](https://github.com/paritytech/polkadot/releases).
```
Comment (carlosala) [inline temp_guide_to_setup_a_node_validator.md:10]:
```suggestion
- The latest version of the Polkadot binary installed on your server. You can download it from [the official Polkadot repository](https://github.com/paritytech/polkadot-sdk/releases).
```

File: temp_guide_to_setup_a_node_validator.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Setting Up a Node/Validator for Paseo Testnet
+
+Welcome to the Paseo testnet! This guide will walk you through the process of setting up a node and becoming a validator on the Paseo testnet network using the Polkadot binary, which is also used for Polkadot and Kusama validators.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A server with a recommended minimum of 8GB RAM, a quad-core CPU, and at least 200GB of free disk space.
```
Comment (carlosala) [inline temp_guide_to_setup_a_node_validator.md:9]: I do not remember which were the specs, if they are this perfect 👍🏻

**Review by carlosala — APPROVED** [review]

File: temp_guide_to_setup_a_node_validator.md (hunk 2 above)
Comment (hbulgarini) [reply temp_guide_to_setup_a_node_validator.md:9]: Actually no, these specs are not right. We have a PAS that i think we could refer here:

https://github.com/paseo-network/paseo-action-submission/blob/main/pas/Hardware_specs.md

Comment (carlosala) [reply temp_guide_to_setup_a_node_validator.md:9]: Let's change them 👍🏻

Comment (educlerici-zondax) [reply temp_guide_to_setup_a_node_validator.md]: already updated @hbulgarini @carlosala please take a look 🙏

**Pull request merged**